unison-src/transcripts/alias-many.output.md

```diff
@@ -41,285 +41,298 @@ Let's try it!
-  125. Nat.eq : Nat -> Nat -> Boolean
-  126. Nat.fromText : Text -> Optional Nat
-  127. Nat.gt : Nat -> Nat -> Boolean
-  128. Nat.gteq : Nat -> Nat -> Boolean
-  129. Nat.increment : Nat -> Nat
-  130. Nat.isEven : Nat -> Boolean
-  131. Nat.isOdd : Nat -> Boolean
-  132. Nat.leadingZeros : Nat -> Nat
-  133. Nat.lt : Nat -> Nat -> Boolean
-  134. Nat.lteq : Nat -> Nat -> Boolean
-  135. Nat.mod : Nat -> Nat -> Nat
-  136. Nat.or : Nat -> Nat -> Nat
-  137. Nat.popCount : Nat -> Nat
-  138. Nat.pow : Nat -> Nat -> Nat
-  139. Nat.shiftLeft : Nat -> Nat -> Nat
-  140. Nat.shiftRight : Nat -> Nat -> Nat
-  141. Nat.sub : Nat -> Nat -> Int
-  142. Nat.toFloat : Nat -> Float
-  143. Nat.toInt : Nat -> Int
-  144. Nat.toText : Nat -> Text
-  145. Nat.trailingZeros : Nat -> Nat
-  146. Nat.xor : Nat -> Nat -> Nat
-  147. type Optional a
-  148. Optional.None : Optional a
-  149. Optional.Some : a -> Optional a
-  150. builtin type Request
-  151. type SeqView a b
-  152. SeqView.VElem : a -> b -> SeqView a b
-  153. SeqView.VEmpty : SeqView a b
-  154. unique type Test.Result
-  155. Test.Result.Fail : Text -> Result
-  156. Test.Result.Ok : Text -> Result
-  157. builtin type Text
-  158. Text.!= : Text -> Text -> Boolean
-  159. Text.++ : Text -> Text -> Text
-  160. Text.drop : Nat -> Text -> Text
-  161. Text.empty : Text
-  162. Text.eq : Text -> Text -> Boolean
-  163. Text.fromCharList : [Char] -> Text
-  164. Text.fromUtf8 : Bytes -> Either Text Text
-  165. Text.gt : Text -> Text -> Boolean
-  166. Text.gteq : Text -> Text -> Boolean
-  167. Text.lt : Text -> Text -> Boolean
-  168. Text.lteq : Text -> Text -> Boolean
-  169. Text.size : Text -> Nat
-  170. Text.take : Nat -> Text -> Text
-  171. Text.toCharList : Text -> [Char]
-  172. Text.toUtf8 : Text -> Bytes
-  173. Text.uncons : Text -> Optional (Char, Text)
-  174. Text.unsnoc : Text -> Optional (Text, Char)
-  175. type Tuple a b
-  176. Tuple.Cons : a -> b -> Tuple a b
-  177. type Unit
-  178. Unit.Unit : ()
-  179. Universal.< : a -> a -> Boolean
-  180. Universal.<= : a -> a -> Boolean
-  181. Universal.== : a -> a -> Boolean
-  182. Universal.> : a -> a -> Boolean
-  183. Universal.>= : a -> a -> Boolean
-  184. Universal.compare : a -> a -> Int
-  185. bug : a -> b
-  186. builtin type crypto.HashAlgorithm
-  187. crypto.HashAlgorithm.Blake2b_256 : HashAlgorithm
-  188. crypto.HashAlgorithm.Blake2b_512 : HashAlgorithm
-  189. crypto.HashAlgorithm.Blake2s_256 : HashAlgorithm
-  190. crypto.HashAlgorithm.Sha2_256 : HashAlgorithm
-  191. crypto.HashAlgorithm.Sha2_512 : HashAlgorithm
-  192. crypto.HashAlgorithm.Sha3_256 : HashAlgorithm
-  193. crypto.HashAlgorithm.Sha3_512 : HashAlgorithm
-  194. crypto.hash : HashAlgorithm -> a -> Bytes
-  195. crypto.hashBytes : HashAlgorithm -> Bytes -> Bytes
-  196. crypto.hmac : HashAlgorithm -> Bytes -> a -> Bytes
-  197. crypto.hmacBytes : HashAlgorithm
+  24.  builtin type Code
+  25.  Code.cache_ : [Tuple Term Code] ->{IO} [Term]
```

Comment:
Noting this signature is off based on earlier comment on signature given in `Builtin.hs`.